packages/date-picker/test/fullscreen.test.js

```diff
@@ -108,20 +108,6 @@ describe('fullscreen mode', () => {
     });
   });
 
-  describe('focused attribute', () => {
-    it('should keep focused attribute after opening overlay', async () => {
```

Comment:
After I removed the explicit `setFocused(true)` from `DelegateFocusMixin.focus()`, this test started to fail with not being able to focus the date-picker's input element . However, it's important to note this test is executed on small screens where the input element isn't focusable by design. With that, the test appears to be generally incorrect.